### ISSUE TITLE
Over time behavior action interpreted incorrectly

### ISSUE THREAD

**Issue by lcdr** [LUDevNet/lu-explorer#106]
We've been interpreting the `OverTime` behavior's `action` key as referring to a behavior so far, but new findings indicate that it actually refers to a skill. Gotta update LU-X's behavior view to reflect this.

**Comment by Xiphoseer** [comment]
Yeah, I think this is only a small change for LU-Explorer (we might want to make the link clickable), but it's a bigger change for LUDevNet/ParadoxServer because that currently always treats `action` as a behavior for the API

**Comment by Xiphoseer** [comment]
TODO

- action on ClearTarget
- miss action on TacArc
